deploy/cr.yaml

```diff
@@ -123,6 +123,9 @@ spec:
 #                - e2e-az1
 #                - e2e-az2
 
+    expose:
```

Comment:
please comment it by default

Reply:
Should we comment it if by default it is exposed via ClusterIP (there is no enable/disable option)?

Reply:
I have commented it out for the moment, because all others are commented as well.